tools/with_lk.sh

```diff
@@ -18,46 +21,37 @@ cat > "$KITE_USERS_FILE" <<EOF
   "isAdmin" : false
 } ]
 EOF
-KITE_DEPLOYMENT_CONFIG_DIR="$(dirname $0)/../test"
-KITE_DEPLOYMENT_CONFIG_DIR=$(realpath "${KITE_DEPLOYMENT_CONFIG_DIR}")
-export HTTP_PORT=$[ 9100 + RANDOM % 100 ]
-export HTTPS_PORT=$[ 9200 + RANDOM % 100 ]
+export KITE_HTTP_PORT=$[ 9100 + RANDOM % 100 ]
```

Comment:
Why did we need to make these ports random at the first place?

Reply:
> Why did we need to make these ports random at the first place?

I think it's to make it more convenient for local testing. You usually run a LynxKite on port 2200 that you're working with. You can use that for testing. This script is for starting a different, brand new LynxKite just for the test.

For testing in Earthly it doesn't matter. A fixed port would be fine.